src/primitives/block.h

```diff
@@ -78,7 +78,7 @@ class CBlock : public CBlockHeader
     std::vector<CTransaction> vtx;
 
     // memory only
-    mutable std::vector<uint256> vMerkleTree;
+    mutable bool fChecked;
```

Comment:
Ideally we'd have this `fChecked` status (which isn't used in CBlock itself) on an administrative object that wraps a CBlock, instead of on CBlock itself.